spring-cloud-dataflow-shell-core/src/test/java/org/springframework/cloud/dataflow/shell/command/TaskCommandTemplate.java

```diff
@@ -81,8 +81,20 @@ public void  launch(String taskName) {
 
 		long value = (long) result.getModel().getValue(1, 1);
 		assertTrue(cr.toString().contains("with execution id " + value));
+		return value;
 	}
 
+	/**
+	 * Stop a task execution.
+	 *
+	 * @param id the id of a {@link org.springframework.cloud.task.repository.TaskExecution}
```

Comment:
twice:

> id the id of a  ....

Reply:
never mind; "id" is the param and what follows is just description - having no delimiter in between caused the confusion - please ignore.